File: Day 040/classwork/code/codewars1.py
"""1) დასქრინეთ ამოხსნილი codewars-ის ამოცანები და ერთად ჩააგდეთ"""
"""https://www.codewars.com/kata/56bc28ad5bdaeb48760009b0/train/python"""
def remove_char(s: str) -> str:
    return s[1:-1]

"""https://www.codewars.com/kata/59a9919107157a45220000e1/train/python"""
def find_all(arr, n):
    result = []
    for i in range(len(arr)):
        if arr[i] == n:
            result.append(i)
    return result

File: day 84/classwork/code/codewars1.js
// https://www.codewars.com/kata/52fba2a9adcd10b34300094c/train/javascript

function transpose(matrix) {
  let result = [];

  for (let i = 0; i < matrix[0].length; i++) {
    result[i] = []; 

    for (let mat = 0; mat < matrix.length; mat++) {
      result[i][mat] = matrix[mat][i];
    }
  }

  return result;
}
